### ISSUE TITLE
Fix broken tests

### ISSUE THREAD

**Issue by hevrard** [google/gapid#3018]
@AWoloszyn commit 838964b introduced the following failures in integration tests, can you have a look?

```
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:277:17: undefined: BoolˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:301:17: undefined: BoolˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:624:17: undefined: CharˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:648:17: undefined: CharˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:963:17: undefined: CharᵖˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:992:17: undefined: CharᵖˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:1307:17: undefined: CharᶜᵖˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:1336:17: undefined: CharᶜᵖˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:1647:17: undefined: F32ˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:1671:17: undefined: F32ˢTypeIndex
bazel-out/k8-fastbuild/bin/gapis/api/test/api_slices.go:1671:17: too many errors
```

**Comment by hevrard** [comment]
Fixed by #3020 (I think)

**Comment by hevrard** [comment]
Actually, this one is still failing, @AWoloszyn any idea?

**Comment by AWoloszyn** [comment]
3038 I think will fix this.

**Comment by hevrard** [comment]
I confirm it does! Thanks @RenfengLiu !
Fixed by #3038
